corehq/apps/reports/templates/reports/partials/save_reports_modal.html

```diff
@@ -48,6 +48,7 @@ <h3 data-bind="text: modalTitle"></h3>
                                 <option value="last7">{% trans "Last 7 days" %}</option>
                                 <option value="last30">{% trans "Last 30 days" %}</option>
                                 <option value="lastmonth">{% trans "Last month" %}</option>
+                                <option value="lastyear">{% trans "Last year" %}</option>
```

Comment:
does this make it easy for someone to setup a report (e.g. "daily form activity") that has a ton of data and impacts server performance?

not necessarily saying we need to do something to change that if it does (was already a hole/possible with a custom range) but it would be nice if we could disable this option for some reports